cookbook/src/tx_taproot.md

```diff
@@ -80,11 +76,11 @@ fn receivers_address() -> Address {
 
 `receivers_address` generates a receiver address.
 In a real application this would be the address of the receiver.
-We use the method `Address::from_str` to parse the string `"bc1p0dq0tzg2r780hldthn5mrznmpxsxc0jux5f20fwj0z3wqxxk6fpqm7q0va"`[^arbitrary_address] into an address.
-Hence, it is necessary to import the `std::str::FromStr` trait.
+We use the `parse` method on `&str` to parse `"bc1p0dq0tzg2r780hldthn5mrznmpxsxc0jux5f20fwj0z3wqxxk6fpqm7q0va"`[^arbitrary_address] as an address.
 Note that `bc1p0dq0tzg2r780hldthn5mrznmpxsxc0jux5f20fwj0z3wqxxk6fpqm7q0va` is a [Bech32](https://bitcoinops.org/en/topics/bech32/) address.
 This is an arbitrary, however valid, Bitcoin mainnet address.
-Hence we use the `require_network` method to ensure that the address is valid for mainnet.
+Bitcoin applications are usually configured with specific Bitcoin network at the start and use that.
+To prevent mistakes related to people sending satoshis to a wrong network we need to call the `require_network` method to ensure that the address is valid for the network, in our case mainnet.
```

Comment:
```suggestion
To prevent mistakes related to people sending satoshis to the wrong network we need to call the `require_network` method to ensure that the address is valid for the network, in our case mainnet.
```

Reply:
I believe my version is correct. There are more networks than 2 so if one is correct in the context then there are multiple incorrect ones and you're only sending to one of them and it's not known which of them you're sending to so "a" makes sense. "the" would imply there's a specific wrong network but there isn't.

Reply:
Technically I think Kix is right, but I think both articles are acceptable in idiomatic English. (When learning mathematics native English speakers have to explicitly learn that "the" is an explicit claim of uniqueness and has a bunch of semantic content.)

Reply:
Thanks, I'm convinced its correct as is and also that it still sounds weird :)